devenv.bat

```diff
@@ -20,7 +20,7 @@ SET CORECLR_PROFILER_PATH_32=%~dp0\src\Datadog.Trace.ClrProfiler.Native\x86\Debu
 
 rem Limit profiling to these processes only
 SET DATADOG_PROFILER_PROCESSES=ConsoleApp.exe;w3wp.exe;iisexpress.exe;Samples.AspNetCoreMvc2.exe;dotnet.exe;Samples.ConsoleFramework.exe
-SET DATADOG_INTEGRATIONS=%~dp0\integrations.json
+rem SET DATADOG_INTEGRATIONS=%~dp0\integrations.json
```

Comment:
Generally my style is to not leave commented code/script in, and just rely on git history to find it again and adding a task in Trello to remember to do it.